luda-editor/luda-editor-client/src/app/sequence_list/mod.rs

```diff
@@ -108,31 +121,103 @@ impl Entity for SequenceList {
+                                .read_dir(luda_editor_rpc::read_dir::Request {
+                                    dest_path: "sequence".to_string(),
+                                })
+                                .await;
+                            match result {
+                                Ok(response) => {
+                                    let titles: Vec<String> = response
+                                        .directory_entries
+                                        .iter()
+                                        .filter_map(|dirent| match dirent.file_type {
+                                            DirentFileType::File => {
+                                                match dirent.name.ends_with(".json") {
+                                                    true => Some(dirent.name.clone()),
+                                                    false => None,
+                                                }
+                                            }
+                                            _ => None,
+                                        })
+                                        .collect();
+
+                                    namui::event::send(
+                                        SequenceListEvent::SequenceTitlesLoadStateUpdateEvent {
+                                            state: SequenceTitlesLoadState {
+                                                started_at,
+                                                detail:
+                                                    types::SequenceTitlesLoadStateDetail::Loaded {
+                                                        titles,
+                                                    },
+                                            },
+                                        },
+                                    )
+                                }
+                                Err(error) => {
+                                    namui::log(format!("error on read_dir: {:?}", error));
+                                    namui::event::send(
+                                        SequenceListEvent::SequenceTitlesLoadStateUpdateEvent {
+                                            state: SequenceTitlesLoadState {
+                                                started_at,
+                                                detail:
+                                                    types::SequenceTitlesLoadStateDetail::Failed {
+                                                        error,
+                                                    },
+                                            },
+                                        },
+                                    );
+                                }
+                            }
+                        }
+                    });
+                }
+                SequenceListEvent::ScrolledEvent { scroll_y } => {
+                    self.scroll_y = *scroll_y;
```

Comment:
You should make sure that mouse is on sequence list or not. It's ok for now because there are nothing but just sequence list in screen. You don't have to fix it now but keep in mind about it.
Reference : https://github.com/NamseEnt/namseent/blob/master/luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/image_browser/scroll.rs#L129